public/components/common/search/search.tsx

```diff
@@ -206,12 +206,23 @@ export const Search = (props: any) => {
   };
 
   useEffect(() => {
-    if (pollingResult && (pollingResult.status === 'SUCCESS' || pollingResult.datarows)) {
-      // update page with data
-      dispatchOnGettingHis(pollingResult, '');
-      // stop polling
-      stopPolling();
-      setIsQueryRunning(false);
+    if (pollingResult) {
```

Comment:
this search component is only for non async polling query search, you may remove these all actually.

Reply:
removed